src/software/thunderscope/replay/BUILD

```diff
@@ -34,3 +34,14 @@ py_library(
         ":proto_player",
     ],
 )
+
+py_binary(
```

Comment:
With this being another binary that we can `bazel run`, `./tbots.py run thunderscope` selects this target instead of Thunderscope (they both are under `//software/thunderscope`). To fix this, I have updated `./tbots.py` to only look at the bazel target name, as opposed to the entire target path. Could you include this change in your PR so we can continue to launch Thunderscope without any problems?
```diff
diff --git a/src/tbots.py b/src/tbots.py
index 82ae4e49d..2cf24eb9f 100755
--- a/src/tbots.py
+++ b/src/tbots.py
@@ -118,16 +118,20 @@ if __name__ == "__main__":
             ]
         )
     )
-    target, confidence = process.extract(args.search_query, targets, limit=1)[0]
-    target = str(target, encoding="utf-8")
+    # Create a dictionary to map target names to complete bazel targets
+    target_dict = {target.split(b":")[-1]: target for target in targets}
+
+    # Use thefuzz to find the best matching target name
+    most_similar_target_name, confidence = process.extract(args.search_query, list(target_dict.keys()), limit=1)[0]
+    target = str(target_dict[most_similar_target_name], encoding="utf-8")
 
     print("Found target {} with confidence {}".format(target, confidence))
 
     if args.interactive or confidence < THEFUZZ_MATCH_RATIO_THRESHOLD:
         filtered_targets = process.extract(
-            args.search_query, targets, limit=NUM_FILTERED_MATCHES_TO_SHOW
+            args.search_query, list(target_dict.keys()), limit=NUM_FILTERED_MATCHES_TO_SHOW
         )
-        targets = [filtered_target[0] for filtered_target in filtered_targets]
+        targets = [target_dict[filtered_target_name[0]] for filtered_target_name in filtered_targets]
         target = str(iterfzf.iterfzf(iter(targets)), encoding="utf-8")
         print("User selected {}".format(target))

```

Reply:
done.